Formula/s/sile.rb

```diff
@@ -23,6 +22,13 @@ class Sile < Formula
     depends_on "libtool" => :build
   end
 
+  # Needed to workaround upstream source dist snafu, see configure phase
+  if build.stable? && OS.mac?
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
```

Comment:
This should be something like
```ruby
stable do
  on_macos do
    ...
  end
end
```

Reply:
Done.

Reply:
Making the syntax linter happy with that is another story. This version moves a bunch of the cheese in a way that is just going to get reverted later when the workarounds are no longer needed. I prefer my neatly encapsulated version that had the workarounds in their own blocks and not mixed with other code. Are you sure about this?